FUI-410 - Change filename ui-core.js to flow-core.js

dshoulders opened the pull request.

TomFullalove commented:
I found references to ui-core.js in package.json and webpack.npm.config.js.
If we're renaming this file, then should it be renamed in these places too?

dshoulders commented:
> I found references to ui-core.js in package.json and webpack.npm.config.js.
> If we're renaming this file, then should it be renamed in these places too?

webpack.npm.config is not used. It looks like it was an attempt to publish ui-core to an npm package. The reference in package.json was also there for using ui-core as an npm package. I've removed the reference in package.json and removed the entire webpack.npm.config file.
